src/index.html

```diff
@@ -17,6 +17,58 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__icon">
+        <a href="#">
+          <img
+            src="/src/images/logo.png"
+            alt="Moyo Icon"
+            width="40"
+            height="40"
+          />
+        </a>
+      </div>
+      <nav class="header__nav">
+        <ul class="header__list">
+          <li class="header__list_item">
```

Comment:
Use semantic class names that represent the meaning of the content. The class name `header__list_item` should be `header__item` to follow BEM methodology more accurately.